### PR TITLE
[DOC] Mention EuroLinux

### PR DESCRIPTION
**Pull request by aronowski**
Mention [EuroLinux](https://en.euro-linux.com/eurolinux/what-is/) as a supported RHEL replica.

### CODE REVIEW

**Comment by ericzolf** [comment]
Hesitating, I didn't know about this distro, and don't want to list zillions of obscure distros, just because they happen to exist. Do you have some numbers/arguments for me?

**Comment by aronowski** [comment]
Naturally!

The company behind the distribution has a way faster release cycle than let's say Rocky. We can compare them on Distrowatch:
- https://distrowatch.com/table.php?distribution=eurolinux

Feature | 9.0 | 8.6
-- | -- | --
Release Date | 2022-06-14 | 2022-05-11

- https://distrowatch.com/table.php?distribution=rocky

Feature | 9.0 | 8.6
-- | -- | --
Release Date | 2022-07-14 | 2022-05-16

EuroLinux usually releases one business day after Red Hat does

dw-eurolinux-8-6

EuroLinux is also in the TOP 100 list on Distrowatch. That's got to mean something.

File: README.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ sudo apt update
 sudo apt install rdiff-backup
 ----
 
-=== CentOS and RHEL 7 (from EPEL)
+=== CentOS, EuroLinux and RHEL 7 (from EPEL)
```
Comment (ikus060) [inline README.adoc:54]: I would simply rephrase that to "RedHat based distribution". No need to list all the RedHat distribution that exists.

And do something similar for Debian, Ubuntu, etc.: "Debian based distribution"

Comment (ericzolf) [reply README.adoc:54]: I like the term "RHEL Replica" used by @aronowski , it differentiates Centos & Co from Fedora, but else I agree on the principle.

Side note: Red Hat, with a space in-between.

Comment (ikus060) [reply README.adoc:54]: Good for me.

Comment (ericzolf) [reply README.adoc:54]: @aronowski do you mind updating your PR accordingly? You could link to https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives (I'm also fine with "RHEL derivatives" BTW, to be consistent).

Comment (aronowski) [reply README.adoc]: @ericzolf Done. Thank you!

**Comment by ericzolf** [comment]
Thanks for the effort. Merging...

**Pull request merged**